### PR TITLE
Get property by ID, rather than by name

### LINKED ISSUE #395: Fix routes properties/name to use an unique identifier
**Describe the bug**
The properties/name routes should use id instead of name OR enforce an unique name for PropertyModel
Because, the name is not unique in the PropertyModel and the find_by_name method only return the first row if multiple property has the same name. There is a risk of get/update/delete the wrong property.
https://github.com/codeforpdx/dwellinglybackend/blob/fbff27efaa80ce29d1cf260b14c0fa560b37b5b8/models/property.py#L58

see 
```
# | GET    | `v1/property/:name`  | Gets a single property     |
# | PUT    | `v1/property/:name`  | Updates a single property  |
# | DELETE | `v1/property/:name`  | Deletes a single property  |
```

**Expected behavior**
We should get an unique identifier to ensure an unique property is being get/update/delete.

### PR DESCRIPTION
**Pull request by gussmith23**
### What issue is this solving?
Closes codeforpdx/dwellingly-app/issues/395

### Any helpful knowledge/context for the reviewer?
Is a re-seeding of the database necessary? ❌ 
Any new dependencies to install? ❌ 
Any special requirements to test? ❌ 

### Please make sure you've attempted to meet the following coding standards
- [ ] Code builds successfully
- [ ] Code has been tested and does not produce errors
- [ ] Code is readable and formatted
- [ ] There isn't any unnecessary commented-out code

### CODE REVIEW

**Comment by NickSchimek** [comment]
Hey @gussmith23 Is this PR ready for me to review?

File: resources/property.py [hunk 1]
```diff
@@ -8,10 +8,10 @@
...
-# | GET    | `v1/properties/`   | Gets all properties        |
-# | GET    | `v1/property/:name`  | Gets a single property     |
-# | PUT    | `v1/property/:name`  | Updates a single property  |
-# | DELETE | `v1/property/:name`  | Deletes a single property  |
+# | GET    | `v1/properties/`     | Gets all properties        |
+# | GET    | `v1/property/:id`    | Gets a single property     |
+# | PUT    | `v1/property/:id`    | Updates a single property  |
+# | DELETE | `v1/property/:id`    | Deletes a single property  |
 
 #TODO Add Id based identifiers.
```
Comment (NickSchimek) [inline resources/property.py:15]: We could probably remove this TODO statement here.

**Review by NickSchimek — DISMISSED** [review]
Looks great! Thank you @gussmith23

**Review by NickSchimek — APPROVED** [review]
I rebased this. so approving one more time.

**Comment by gussmith23** [comment]
Argh, sorry I was AWOL. I would be surprised if this doesn't break things now that it's merged...wonder if anyone has hit problems on the frontend?

**Comment by NickSchimek** [comment]
I probably merged this too soon. However, there is a PR to update the frontend code. codeforpdx/dwellingly-app/pull/493

**Pull request merged**